### PR TITLE
✨ Source Typeform: Migrate to Low Code

### LINKED ISSUE #28121: typeform

### PR DESCRIPTION
**Pull request by topefolorunso**
## What
Migrating Source Typeform to Low-Code CDK resolves #28121

## How
Created the connector using the connector buider UI, then imported the manifest.yaml to the low code cdk.

## Recommended reading order
1. `manifest.yaml`

## 🚨 User Impact 🚨
*Are there any breaking changes? What is the end result perceived by the user?*

*For connector PRs, use this section to explain which type of semantic versioning bump occurs as a result of the changes. Refer to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) guidelines for more information. **Breaking changes to connectors must be documented by an Airbyte engineer (PR author, or reviewer for community PRs) by using the [Breaking Change Release Playbook](https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit).***

*If there are breaking changes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.*

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-typeform/source_typeform/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,512 @@
...
+      requester:
+        type: HttpRequester
+        url_base: https://api.typeform.com/
+        path: forms/{{ stream_partition.form_id }}
+        http_method: GET
+        request_parameters: {}
+        request_headers: {}
+        authenticator:
+          type: BearerAuthenticator
+          api_token: '{{ config[''api_key''] }}'
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-typeform/source_typeform/manifest.yaml:23]: Current connector has two type of auth. You need to implement the Oauth too.

File: airbyte-integrations/connectors/source-typeform/source_typeform/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,512 @@
...
+      type: SimpleRetriever
+      requester:
+        type: HttpRequester
+        url_base: https://api.typeform.com/
+        path: themes
+        http_method: GET
+        request_parameters: {}
+        request_headers: {}
+        authenticator:
+          type: BearerAuthenticator
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-typeform/source_typeform/manifest.yaml:360]: This doesn't correspond to what the connector spec is.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Fix comments. Connector has two type of auth.

**Comment by topefolorunso** [comment]
Fixed now @marcosmarxm, please review

File: airbyte-integrations/connectors/source-typeform/source_typeform/manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+        $ref: "#/definitions/no_selector"
+      partition_router:
+        $ref: "#/definitions/form_id_partition_router"
+
+  responses_stream:
+    type: DeclarativeStream
+    name: responses
+    primary_key: response_id
+    retriever:
+      type: SimpleRetriever
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-typeform/source_typeform/manifest.yaml:100]: The connector is almost working. It is failing for a incremental test. We're going to return with more info during the week @topefolorunso

File: airbyte-integrations/connectors/source-typeform/source_typeform/components.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+        if form_ids:
+            for item in form_ids:
+                yield {"form_id": item}
+        else:
+            for parent_stream_config in self.parent_stream_configs:
+                print(parent_stream_config)
+                for item in parent_stream_config.stream.read_records(sync_mode=SyncMode.full_refresh):
+                    yield {"form_id": item["id"]}
+
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-typeform/source_typeform/components.py:42]: PLease add some unit test for this function and remove the print statement (I added it :P)

**Review by marcosmarxm — DISMISSED** [review]
Some comments.

File: airbyte-integrations/connectors/source-quickbooks/source_quickbooks/manifest.yaml [hunk 5]
```diff
@@ -27,6 +27,7 @@ definitions:
       client_secret: "{{ config['credentials']['client_secret'] }}"
       refresh_token: "{{ config['credentials']['refresh_token'] }}"
       refresh_token_updater: {}
+      grant_type: refresh_token
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-quickbooks/source_quickbooks/manifest.yaml:30]:
```suggestion
      grant_type: refresh_token
```

**Review by sh4sh — APPROVED** [review]

**Comment by sh4sh** [comment]
Thank you for the contribution!

**Pull request merged**